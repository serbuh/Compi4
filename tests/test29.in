// Declaration and assignments with computations

K x;
C y = 20 C;
F z = 500 K;
x = 50 F;

C v = x + y + 50 F;
C vv = v + z/2;

K xx;
K yy;
xx = x*5;
yy = y*5;
C zz = xx + yy;

print v;
print vv;
print xx;
print yy;
print zz;
