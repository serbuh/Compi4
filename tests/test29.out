586
690
1415
373
1515
